01_P/P_2_1_1_03/sketch.js

```diff
@@ -45,22 +46,18 @@ var alphaRight = 100;
 
 var actRandomSeed = 0;
 
-
 function setup() {
   createCanvas(600, 600);
-
-  colorWhite = color(255);
-
   colorMode(HSB, 360, 100, 100, 100);
+  noFill();
 
+  colorWhite = color(255);
   colorRight = color(0, 0, 0, alphaRight);
   colorLeft = color(323, 100, 77, alphaLeft);
 }
 
 function draw() {
   background(colorWhite);
-  smooth();
```

Comment:
Removed `smooth()` since this is already enabled by default in every p5 sketch (as well as Processing 2 and 3).